examples/demo.js

```diff
@@ -211,6 +214,18 @@ window.onload = function () {
     download(image, 'index.gif', 'image/gif');// downloadjs library function
   }
 
+  function savetopubliclab(){
+    function postToPL(imgSrc) {
+      var uniq = Date.now();
+      $('body').append('<form method="post" id="postToPL' + uniq + '" action="https://publiclab.org/post" target="postToPLWindow"><input type="hidden" name="datauri_main_image" /></form>');
+      f = $('#postToPL' + uniq)[0];
+      f.datauri_main_image.value = imgSrc;
+      window.open('', 'postToPLWindow');
+      f.submit();
+    }
+    postToPL($('img')[sequencer.steps.length - 1].src);
+  }
```

Comment:
Is there any benefit to nesting this function? You could do something like the following to remove the extra function call and the local variable:
```
function saveToPublicLab() {
    var uniq = Date.now();
    $('body').append('<form method="post" id="postToPL' + uniq + '" action="https://publiclab.org/post" target="postToPLWindow"><input type="hidden" name="datauri_main_image" /></form>');
    f = $('#postToPL' + uniq)[0];
    f.datauri_main_image.value = $('img')[sequencer.steps.length - 1].src;
    window.open('', 'postToPLWindow');
    f.submit();
}
```